ubccr/xdmod
# Updating Backup Tables to not be Temporary

## Description
The backup tables will now be fully fledged tables instead of temporary tables.
This will ensure that if the `acl-config` process stops before completion, but
after the backups have been performed, these tables will still be available to
reconstruct their source tables.

## Motivation and Context
Slightly more robust than the previous implementation

## Tests performed
Manual Tests were performed:

Testing Environments:
  - Open XDMoD Fresh Install
  - Open XDMoD Upgrade
  - XSEDE upgrade

Test:
  - Spin up docker && install / upgrade as appropriate
  - `mysql` and check contents of the source tables and `bkup` tables. Ensure that they contain the same number of records and the values make sense. 

## Types of changes


## Checklist:
